src/pages/Task/index.tsx

```diff
@@ -340,6 +351,40 @@ const Task: React.FC<TaskViewProps> = ({
                       },
                     ]
                   : []),
+                // Render cards at the end of sections if enabled.
+                ...(FEATURE_FLAGS.CARDS && cards.status === 'Ok' && cards.data
+                  ? cards.data.map((def) => ({
+                      key: def.hash,
+                      order: 99,
```

Comment:
This is to make sure that cards are rendered last in the list. I believe that it does not matter if we have multiple items with same order since .sort should always return list in same way.